chapter2/tests/t2/chapter2-t2-gen-kustomization.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# Processes repositories list, generates the ed25519 key for each of them and
+# outputs the Kustomization configuration.
+set -eo pipefail
+
+readonly DEPLOY_KEY_POSTFIX="deploy-key.pem"
+readonly REPO_DIR="repos"
+readonly KUSTOMIZATION_FILE="./$REPO_DIR/kustomization.yaml"
+
+if [ $# -eq 0 ]; then
+  echo "Usage: chapter2-t2-gen-kustomization.sh [REPO]..."
```

Comment:
Actually, script file name is also an argument you pass to the shell, so you can do this
```suggestion
  echo "Usage: $(basename "$0") [REPO]..."
```
Just in case the user renames a file